robotis-op2/accelerometer.md

```diff
@@ -3,19 +3,16 @@
 This tab can be used to investigate the values of the accelerometer while the
 controller is running. If the checkbox is checked, the values of the
 accelerometer are shown and plotted on the graph in real time. Four different
-types of graph can be plot. The first three are one axis in function of an
+types of graph can be plotted. The first three are one axis in function of an
 other, and the last one, plots the value of the three axes in function of the
 time. The corresponding colors are the following:
 
 - Red for axis X
 - Green for axis Y
 - Blue for axis Z
 
-%figure "Accelerometer tab of the robot-window"
+%figure "Accelerometer tab of the Robot Window"
```

Comment:
Right, I will open another PR to make other books consistent.
I think using capital letters is good teaching method, because it emphasizes the Webots concept.